src/js/App/RootApp.js

```diff
@@ -17,6 +18,24 @@ import useOuiaTags from '../utils/useOuiaTags';
 import Banner from './Banners/Banner';
 import cookie from 'js-cookie';
 
+const QuickStartCatalog = lazy(() =>
+  import(/* webpackChunkName: "quick-start" */ './QuickStartCatalog').then(({ QuickStartCatalog }) => ({ default: QuickStartCatalog }))
```

Comment:
We usually use default exports. Could you change the `QuickStartCatalog.js` file to use default export and remove the `then` branch?